CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
+
+### Release Notes
+* Features:
+  * Kubernetes 1.33 version is now available for Preview, see [Release tracker](https://releases.aks.azure.com/) for when it hits your region.
+  * [Network isolated cluster](https://learn.microsoft.com/azure/aks/concepts-network-isolated) with outbound type `none` is now Generally Available.
+  * [AKS Security Bulletin](https://learn.microsoft.com/azure/aks/security-bulletins/overview?tabs=aks-addons%2Caks-cluster%2Caks-node-image) and [AKS CVE Mitigation Status](https://releases.aks.azure.com/webpage/index.html) are now available to track Security and CVE mitigations.
+
+* Bug Fixes:
+  * Fix an [issue](https://github.com/azure-networking/cilium-private/pull/465)) in [Azure CNI Powered by Cilium](https://learn.microsoft.com/azure/aks/azure-cni-powered-by-cilium), if the DNS request timeout is set to below 2 seconds, the high-scale scenarios may cause drops due to duplicate request ID.
+  * Fix an issue where [load balancer](https://learn.microsoft.com/azure/aks/load-balancer-standard) tags were not updated after a managed cluster tag update. Load balancer tags now correctly reflect the latest state.
+  * Fix an [issue](https://github.com/cilium/cilium/issues/38210) in Cilium v1.17 where a deadlock causing pods unable to start. 
+
+* Behavior Changes:
+  * `aksmanagedap` is blocked as a reserved name for AKS system component, you can no longer use it for creating agent pool.
+  * `linuxutil plugin` is temporarily disabled for [Retina Basic and ACNS](https://retina.sh/docs/Metrics/plugins/Linux/linuxutil) as it is causing memory leaks that leads to Retina pods OOMKill.
+  * ACNS configmaps (`cilium`, `retina`, `hubble`) now auto‑format cluster names to satisfy Cilium 1.17 rules:≤ 32 chars, lowercase alphanumeric characters and dashes, no leading/trailing dashes, functionality is unaffected. This change is due to the strict enforcement of Cilium 1.17. See this [link](https://github.com/cilium/cilium/commit/b1f10bcb26d71f046d8ad108d5f4b9e42aa5d486) for details.
+  * The `defaultConfig.gatewayTopology` field is now included in the Istio add-on `MeshConfig` AllowList as an unsupported field. For more details, see the [Istio MeshConfig documentation](https://learn.microsoft.com/azure/aks/istio-meshconfig#allowed-supported-and-blocked-meshconfig-values).
+  * Previously, you can't disable [NodeAutoProvisioning](https://learn.microsoft.com/azure/aks/node-autoprovision) once enabled, now you can if meet these criteria:
+    - The cluster must have 0 NAP-managed nodes.
+    - Every existing karpenter.sh/NodePool must have its spec.limits.cpu set to 0.
```

Comment:
All of this is docs info not release notes info, provide doc link to the expanded section after noting you can disable nap.

Reply:
Tagging Wilson offline, keep open for now

Reply:
@wilson to add disable criteria in NAP doc